### PR TITLE
Add list target unit test

### PR DESCRIPTION
**Pull request by sidpremkumar**
## Description of the change

Adds list target unit test to test all types of filters + general list targets

This PR also refactors our unit test directory and tests

Needs to be merged + rebased after this: https://github.com/bastionzero/zli/runs/5978478117?check_suite_focus=true

## Relevant release note information

Release Notes:

## Related JIRA tickets

Relates to JIRA: CWC-1686

## Relevant HotFix if applicable

Relevant HotFix PR Number:

## Have you considered the security impacts?

Does this PR have any security impact?

- [ ] Yes
- [X] No

If yes, please explain:

### CODE REVIEW

File: src/unit-tests/tests/utils/utils.test.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        const validSSMTargetTypeStrings = [
+            'ssmtarget',
+            'SSMtarget',
+            'sSMtarget'
+        ];
+        validSSMTargetTypeStrings.forEach(t => expect(parseTargetType(t)).toBeDefined());
+    });
+    
+    test('invalid targetType strings', () => {
+        const validSSMTargetTypeStrings = [
```
Comment (chad-cwc) [inline src/unit-tests/tests/utils/utils.test.ts:14]: Rename to `invalidSSMTargetTypeStrings`?

File: src/unit-tests/tests/utils/utils.test.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            'ssm-user@neat-test:/hello', // valid path
+            'ssm-user@coolBeans:::', // everything after first colon ignored
+            'ssm-user@97d4d916-33f8-478e-9e6c-1091662ccaf0:asdfjl; asdfla;sd',
+            '97d4d916-33f8-478e-9e6c-1091662ccaf0:asdfjl; asdfla;sd'
+        ];
+        validSSMTargetStrings.forEach(t => expect(parseTargetString(t)).toBeDefined());
+    });
+    
+    test('invalid targetStrings', () => {
+        const validSSMTargetStrings = [
```
Comment (chad-cwc) [inline src/unit-tests/tests/utils/utils.test.ts:39]: Rename to `invalidSSMTargetStrings`?

**Review by chad-cwc — APPROVED** [review]
Couple nits that I'll let you decide if you want to change, but LGTM.

File: src/unit-tests/tests/utils/utils.test.ts (hunk 2 above)
Comment (sidpremkumar) [reply src/unit-tests/tests/utils/utils.test.ts]: https://github.com/bastionzero/zli/pull/369/commits/f986a2bec8251bf1b8154297883f7bbeca4aa9e5

File: src/unit-tests/tests/utils/utils.test.ts (hunk 1 above)
Comment (sidpremkumar) [reply src/unit-tests/tests/utils/utils.test.ts]: https://github.com/bastionzero/zli/pull/369/commits/f986a2bec8251bf1b8154297883f7bbeca4aa9e5

**Pull request merged**